app/src/main/java/com/hover/stax/requests/Request.java

```diff
@@ -123,7 +122,7 @@ String generateWhatsappRecipientString(List<StaxContact> contacts, Channel c) {
 
     public String generateMessage(Context c) {
         String amountString = amount != null ? c.getString(R.string.sms_amount_detail, Utils.formatAmount(amount)) : "";
-        String noteString = note != null && !TextUtils.isEmpty(note) ? c.getString(R.string.sms_note_detail, note) : "";
+        String noteString = note != null && !TextUtils.isEmpty(note) ? c.getString(R.string.sms_note_detail, note) : " ";
```

Comment:
I don't think we want an empty space when the note is empty?

Reply:
Yeah. I shouldn't be empty.